README.md

```diff
@@ -1,20 +1,28 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
+it's a function that defines what happens when an object is clicked
 
-(b) Where in your JS file should you put it, and WHY? 
+(b) Where in your JS file should you put it, and WHY?
+someObject.on('click', clickHandler); <-- or have the function embeded right in the method
```

Comment:
You wrote the click handler correctly, but the question was where should it go. Answer: in the `$(document).ready()` because the HTML needs to load first before jQuery can look for the element ID.